OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -601,6 +601,30 @@ internal static void UpgradeActorRules(ModData modData, int engineVersion, ref L
 					if (node.Key == "SoundFile" && parent.Key.StartsWith("AmbientSound", StringComparison.Ordinal))
 						RenameNodeKey(node, "SoundFiles");
 
+				// PauseOnLowPower property has been replaced with PauseOnCondition/RequiresCondition
+				if (engineVersion < 20170430)
```

Comment:
Can you please also print a message about rewiring `AffectedByPowerOutage` using conditions if it is found on an actor?